### PR TITLE
Upgrade maven-plugin-plugin and maven-shade-plugin

### PR DESCRIPTION
**Pull request by slachiewicz**
Fixes problem with compilation for bytecode 9+

### CODE REVIEW

**Comment by slachiewicz** [comment]
Yes, i'll work on that.

File: org.jacoco.build/pom.xml [hunk 1]
```diff
@@ -734,14 +734,6 @@
       </build>
     </profile>
 
-    <!--
-    Following profiles enable compilation into bytecode version 9
-    when requested "bytecode.version" greater than 9,
-    because this is maximum that can be processed by
-    maven-shade-plugin (https://issues.apache.org/jira/browse/MSHADE-289)
```
Comment (Godin) [inline org.jacoco.build/pom.xml]: [First builds of JDK 13 EA already available for testing](http://jdk.java.net/13/) and I think same trick will be needed to test new bytecode version, but ok we can restore comment and add profile when will be needed.

**Comment by marchof** [comment]
@Godin Do the changed profiles have an impact on your bytecode version matrix?

    org.jacoco.doc/docroot/doc/build.html

**Comment by Godin** [comment]
@marchof this matrix shows defaults, so no impact 😉

**Pull request merged**